services/server/src/server/services/utils/database-util.ts

```diff
@@ -171,11 +179,11 @@ ${
 
 export async function insertCode(
   pool: Pool,
-  { bytecode_hash, bytecode }: Tables.Code,
+  { bytecode_hash, bytecode_hash_keccak, bytecode }: Tables.Code,
 ) {
   await pool.query(
-    "INSERT INTO code (code_hash, code) VALUES ($1, $2) ON CONFLICT (code_hash) DO NOTHING",
-    [bytecode_hash, bytecode],
+    "INSERT INTO code (code_hash, code_hash_keccak, code) VALUES ($1, $2, $3) ON CONFLICT (code_hash) DO NOTHING",
```

Comment:
I don't think you can instert `code_hash`? That was the whole point of the changes, no? 

Also IMO that means, we should get the `code_hash` from the DB and not calculate ourselves on our server.